dockstore-webservice/src/main/java/io/dockstore/webservice/core/Tag.java

```diff
@@ -44,6 +45,11 @@
 @SuppressWarnings("checkstyle:magicnumber")
 public class Tag extends Version<Tag> implements Comparable<Tag> {
 
+    @Column
+    @JsonProperty("last_build")
```

Comment:
Curious what others think, because only I may think this way, but I would say `last_built` -- to me it would more clearly imply a date/time, whereas `last_build` indicates more of a version/tag.

Reply:
I think for this we copied the field quay was returning (I might be wrong). Though I agree last build implies a tag and not a time.

Reply:
To me, last build implies a hash, commit id, or some kind of reference to the Docker/container image.